util/annotations/helpers.go

```diff
@@ -49,3 +50,11 @@ func HasWithPrefix(prefix string, annotations map[string]string) bool {
 	}
 	return false
 }
+
+// SetNodeAnnotation sets a key value annotation on the Node.
+func SetNodeAnnotation(node *corev1.Node, key, value string) {
+	if node.Annotations == nil {
+		node.Annotations = make(map[string]string)
+	}
+	node.Annotations[key] = value
+}
```

Comment:
Could this be on a metav1 object instead?

Reply:
indeed, changed